app/src/main/java/com/kms/cura/controller/UserController.java

```diff
@@ -2,15 +2,28 @@
 
 import android.content.Context;
 
+import com.kms.cura.entity.DegreeEntity;
+import com.kms.cura.entity.FacilityEntity;
+import com.kms.cura.entity.SpecialityEntity;
+import com.kms.cura.entity.user.DoctorUserEntity;
 import com.kms.cura.entity.user.UserEntity;
 import com.kms.cura.model.UserModel;
 
+import java.sql.Date;
+import java.util.List;
+
 public class UserController {
     public static void registerPatient(String name, String email, String password) {
         UserEntity entity = new UserEntity(null, name, email, password);
         UserModel.getInstace().registerPatient(entity);
     }
-
+    public static void registerDoctor(String id, String name, String email, String password, String phone, DegreeEntity degree,
+                                      List<SpecialityEntity> speciality, double rating, int experience, double minPrice, double maxPrice,
```

Comment:
should rename variable like speciality and facility to something closer to their type. Either suffix them with "list" or add "s" is fine